### ISSUE TITLE
[WCLIENT 1.0] java.io.IOException: Server returned HTTP response code: 400 for URL: http://wincho.kro.kr/winchoclient/public servers.json [AUTO GENERATED]

### ISSUE THREAD

**Issue by WinChoReportingSystem** [WintChoco/WinchoClient#28]
mouseClicked event handler
```
Description: java.io.IOException: Server returned HTTP response code: 400 for URL: http://wincho.kro.kr/winchoclient/public servers.json
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1985)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1577)
	at java.base/java.net.URL.openStream(URL.java:1192)
	at net.minecraft.class_500.handler$zze000$init(class_500.java:536)
	at net.minecraft.class_500.method_25426(class_500.java:112)
	at net.minecraft.class_437.method_25423(class_437.java:387)
	at net.minecraft.class_310.method_1507(class_310.java:986)
	at net.minecraft.class_442.method_19860(class_442.java:164)
	at net.minecraft.class_4185.method_25306(class_4185.java:40)
	at net.minecraft.class_4264.method_25348(class_4264.java:16)
	at net.minecraft.class_339.method_25402(class_339.java:121)
	at net.minecraft.class_4069.method_25402(class_4069.java:27)
	at net.minecraft.class_442.method_25402(class_442.java:311)
	at net.minecraft.class_312.method_1611(class_312.java:94)
	at net.minecraft.class_437.method_25412(class_437.java:490)
	at net.minecraft.class_312.method_1601(class_312.java:94)
	at net.minecraft.class_312.method_22686(class_312.java:165)
	at net.minecraft.class_1255.execute(class_1255.java:101)
	at net.minecraft.class_312.method_22684(class_312.java:165)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3174)
	at com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:190)
	at net.minecraft.class_310.method_1523(class_310.java:1142)
	at net.minecraft.class_310.method_1514(class_310.java:4156)
	at net.minecraft.client.main.Main.main(Main.java:217)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:234)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:153)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)

```

```
-- System Details --
Details:
	Minecraft Version: 1.17
	Minecraft Version ID: 1.17
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 16.0.1, Amazon.com Inc.
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Amazon.com Inc.
	Memory: 314808912 bytes (300 MiB) / 922746880 bytes (880 MiB) up to 1073741824 bytes (1024 MiB)
	CPUs: 4
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i5-7600 CPU @ 3.50GHz
	Identifier: Intel64 Family 6 Model 158 Stepping 9
	Microarchitecture: Kaby Lake
	Frequency (GHz): 3.50
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 4
	Graphics card #0 name: NVIDIA GeForce GTX 1060 3GB
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 3072.00
	Graphics card #0 deviceId: 0x1c02
	Graphics card #0 versionInfo: DriverVersion=27.21.14.5671
	Memory slot #0 capacity (MB): 4096.00
	Memory slot #0 clockSpeed (GHz): 2.13
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 4096.00
	Memory slot #1 clockSpeed (GHz): 2.13
	Memory slot #1 type: DDR4
	Memory slot #2 capacity (MB): 4096.00
	Memory slot #2 clockSpeed (GHz): 2.13
	Memory slot #2 type: DDR4
	Virtual memory max (MB): 14684.54
	Virtual memory used (MB): 9650.32
	Swap memory total (MB): 2432.00
	Swap memory used (MB): 107.30
	JVM Flags: 8 total; -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=16M -Xmx1024m -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump
	Launched Version: fabric-loader-0.11.6-1.17
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: GeForce GTX 1060 3GB/PCIe/SSE2 GL version 3.2.0 NVIDIA 456.71, NVIDIA Corporation
	Window size: 854x480
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: 
	Current Language: English (US)
	CPU: 4x Intel(R) Core(TM) i5-7600 CPU @ 3.50GHz
```

**Comment by WinChoReportingSystem** [comment]
java.io.IOException: Server returned HTTP response code: 400 for URL: http://wincho.kro.kr/winchoclient/public servers.json
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1985)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1577)
	at java.base/java.net.URL.openStream(URL.java:1192)
	at net.minecraft.class_500.handler$zze000$init(class_500.java:536)
	at net.minecraft.class_500.method_25426(class_500.java:112)
	at net.minecraft.class_437.method_25423(class_437.java:387)
	at net.minecraft.class_310.method_1507(class_310.java:986)
	at net.minecraft.class_442.method_19860(class_442.java:164)
	at net.minecraft.class_4185.method_25306(class_4185.java:40)
	at net.minecraft.class_4264.method_25348(class_4264.java:16)
	at net.minecraft.class_339.method_25402(class_339.java:121)
	at net.minecraft.class_4069.method_25402(class_4069.java:27)
	at net.minecraft.class_442.method_25402(class_442.java:311)
	at net.minecraft.class_312.method_1611(class_312.java:94)
	at net.minecraft.class_437.method_25412(class_437.java:490)
	at net.minecraft.class_312.method_1601(class_312.java:94)
	at net.minecraft.class_312.method_22686(class_312.java:165)
	at net.minecraft.class_1255.execute(class_1255.java:101)
	at net.minecraft.class_312.method_22684(class_312.java:165)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3174)
	at com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:190)
	at net.minecraft.class_310.method_1523(class_310.java:1142)
	at net.minecraft.class_310.method_1514(class_310.java:4156)
	at net.minecraft.client.main.Main.main(Main.java:217)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:234)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:153)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)

**Comment by WintChoco** [comment]
Resolved in WinCho Client 1.0.1
